libraries/AP_HAL_ChibiOS/hwdef/CubeSolo/hwdef.dat

```diff
@@ -46,3 +46,41 @@ COMPASS AK09916:probe_ICM20948 0 ROTATION_ROLL_180_YAW_90
 
 # also probe for external compasses
 define HAL_PROBE_EXTERNAL_I2C_COMPASSES
+
+DMA_NOSHARE USART6_TX USART6_RX USART2_TX ADC1
+
+# telem2 does not need DMA
```

Comment:
serial2 is bared as part of the external expansion port.   It might very well need DMA.

Reply:
same as above